### PR TITLE
Rename all the things to ambiata

### PR DESCRIPTION
**Pull request by charleso**
@markhibberd This what you had in mind? /cc @olorin

### CODE REVIEW

File: ambiata-cli.cabal [hunk 1]
```diff
@@ -59,15 +59,15 @@ library
...
-                       TatooineCli.Downloads
+                       Ambiata
+                       Ambiata.Data
+                       Ambiata.Incoming
+                       Ambiata.Json
+                       Ambiata.Files
+                       Ambiata.Processing
+                       Ambiata.Env
+                       Ambiata.Credentials
+                       Ambiata.Downloads
```
Comment (olorin) [inline ambiata-cli.cabal]: Not sure how likely we are to be publishing other packages in this namespace; might be worth considering `Ambiata.Cli` or something?

**Comment by olorin** [comment]
:shaved_ice:

File: ambiata-cli.cabal (hunk 1 above)
Comment (charleso) [inline ambiata-cli.cabal]: > Not sure how likely we are to be publishing other packages in this namespace; might be worth considering Ambiata.Cli or something?

Yeah I was on the fence too. @nhibberd any thoughts?

Comment (markhibberd) [inline ambiata-cli.cabal]: :100: for `Ambiata.Cli`

File: README.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
-tatooine-cli
-============
+Ambiata Command Line
+====================
 
 
 High level data flows
 ---------------------
```
Comment (markhibberd) [inline README.md]: We should do a second sweep to demote upload specific docs (there will be many siblings at some point I am assuming), and turn this into a how to get & use `ambiata-cli`. Separate issue for later though I think - not that critical right now.

**Comment by markhibberd** [comment]
:+1: looks good. Will rename the repository at some point as well.

**Comment by charleso** [comment]
Have updated to `Ambiata.Cli`. Will merge after a green build.

**Pull request merged**